src/Swoole/Response.php

```diff
@@ -95,6 +95,46 @@ public function send(string $body = '', int $exit = null): void
         }
     }
 
+    /**
+     * Output response
+     *
+     * Generate HTTP response output including the response header (+cookies) and body and prints them.
+     *
+     * @param string $body
+     * @param bool $last
+     *
+     * @return void
+     */
+    public function chunk(string $body = '', bool $last = false): void
```

Comment:
Any chance this should or could be part of utopia/fw i a way that we can keep a consistent interface?

Reply:
```suggestion
    public function chunk(string $body = '', bool $end = false): void
```

Might be more decoupled from the files behavior and a bit more generic.